search-app/src/cmr/search/services/query_execution/facets/facets_v2_results_feature.clj

```diff
@@ -146,18 +146,30 @@
   [_]
   nil)
 
+(defn- get-facet-fields-map
+  "Returns a map with the keys being the keys in facet-fields-list
+   and the values being the related facet size in the facets-size map. 
+   If the value is not present in facets-size-map, use the default value."
+  [facet-fields-list facets-size-map]
+  (let [default-value-list (repeat (count facet-fields-list) DEFAULT_TERMS_SIZE)
```

Comment:
Rather than constructing the `default-facet-feilds-map` programmatically, we should just `def` the `default-facet-feilds-map` as it is something static.

Reply:
What do you mean?  Both default-value-list and default-facet-fields-map depends on the facet-fields-list, which is dynamic.

Reply:
Take a look at `cmr.search.services.query-execution.facets.collection-v2-facets` where `collection-facets-v2-params` are defined through `collection-facets-v2-params->elastic-fields`, you could def a var (e.g. v2-facets-default-size) to list field names and their default sizes.

Reply:
Done.